packages/react/src/Pagination/Pagination.tsx

```diff
@@ -118,36 +121,32 @@ export const Pagination = forwardRef(
       onChangePage(currentPage + 1)
     }
 
+    const id = useId()
+    const navLabelId = `nav-label-${id}`
```

Comment:
We don't need this, right? `useId` already ensures the id is unique?

Reply:
Adjusted